client.go

```diff
@@ -52,6 +52,14 @@ const (
 
 type Timestamp time.Time
 
+// return by Capture for nil clients
+var closedChan chan error
```

Comment:
I get that you're trying to reuse one single object for every case, but I think I'd rather create a unique closed channel for each call. Especially since, in theory, this should be getting used less often. So for the cases where we don't need it, we're allocating unnecessary memory.